### DISCUSSION TITLE
'Error opening file for writing'

### DISCUSSION THREAD

**Discussion by MidnightRetro** [AntiMicroX/antimicrox#1021] [Q&A] [answered]
What am I doing wrong here?
![image](https://github.com/user-attachments/assets/821486ae-a507-4a8d-be80-b6bd5874115b)

**Comment by pktiuk** [comment]
Did this happen during update or installation?      
Did you install it as an admin?

**Reply by nfourbanks** [reply]
That error I had because I had not quit the icon from the start-up. Having done that, it then installed

**Reply by MidnightRetro** [reply]
Yes, during update, and yes, installed as admin

**Reply by MidnightRetro** [reply, marked as answer]
Now it works, I have no idea what I was doing wrong. Maybe it was running in the taskbar?
